cpp/oneapi/dal/algo/subgraph_isomorphism/test/badargs.cpp

```diff
@@ -186,15 +186,6 @@ SUBGRAPH_ISOMORPHISM_BADARG_TEST("Throws if match count is negative") {
         invalid_argument);
 }
 
-SUBGRAPH_ISOMORPHISM_BADARG_TEST("Throws if match count is positive") {
```

Comment:
Just for a check. Do you really want to delete this?

Reply:
Reverted, thank you/